Sassy-Snowmen/reactive-nutshell
# Mj user and friends

# Description

Details button was removed and Friends username displays and is a link. The link does not do anything yet.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. `` git fetch --all``
2. ``git checkout mj-UserAndFriends``
3.  verify that the details button is gone and the user names display correctly


# Checklist:
